Customizing/README.md

```diff
@@ -0,0 +1,68 @@
+# Customizing
+
+<!-- MarkdownTOC depth=0 autolink="true" bracket="round" autoanchor="true" style="ordered" indent="   " -->
+
+1. [Introduction](#introduction)
+1. [User Agreements](#user-agreements)
+1. [System Language Changes](#system-language-changes)
+1. [Skins and Styles](#skins-and-styles)
+
+<!-- /MarkdownTOC -->
+
+<a name="introduction"></a>
```

Comment:
This is true, but this will obviously only work on GitHub, I'm not sure if the TOC-links will still work if you import the file into the ILIAS Wiki (like http://www.ilias.de/docu/goto_docu_pg_75024_367.html)